public/dashboard.css

```diff
@@ -1,33 +1,43 @@
-body {
-  font-family: sans-serif;
-  background-color: #000000;
-  margin: 0.1vh;
-  height: 100vh;
-}
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&family=Roboto+Mono:wght@500&display=swap');
 
-div.dashboard {
-  width: 100%;
-  height: 100%;
+html {
+  --dashboard-radius: 8px;
```

Comment:
I think we should try to avoid using any px units... we don't know the resolutions of the screens we'll be running on, and they may change drastically year-to-year.